dbms/src/Dictionaries/DictionaryStructure.cpp

```diff
@@ -387,4 +399,302 @@ std::vector<DictionaryAttribute> DictionaryStructure::getAttributes(
     return res_attributes;
 }
 
+
+void buildXMLRecursive(
```

Comment:
`buildConfigurationRecurse` or something like this. We just build structured configuration here. It doesn't serialize to xml.